packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -33,7 +33,7 @@ import { TELEMETRY_METADATA_COUNT } from '../constants';
 import { WorkspaceContext, workspaceContextUtils } from '../context';
 import { handleDeployDiagnostics } from '../diagnostics';
 import { nls } from '../messages';
-import { setApiVersion, setSourceApiVersion } from '../services/sdr/componentSetUtils';
+import { componentSetUtils } from '../services/sdr/componentSetUtils';
```

Comment:
Changed the way the functions in componentSetUtils.ts are exported in order to write unit tests for them.